RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc

```diff
@@ -0,0 +1,287 @@
+#include "RecoEgamma/EgammaTools/interface/HGCalShowerShapeHelper.h"
+
+const double HGCalShowerShapeHelper::kLDWaferCellSize_ = 0.698;
+const double HGCalShowerShapeHelper::kHDWaferCellSize_ = 0.465;
+
+HGCalShowerShapeHelper::HGCalShowerShapeHelper(edm::ConsumesCollector &sumes)
+    : caloGeometryToken_{sumes.esConsumes()} {}
```

Comment:
@jpata Can I take the same approach in 11_1?

Reply:
as far as I can see, esConsumes and ConsumesCollector are available in 11_1.

Reply:
@jpata I am probably missing something very simple, but with the same code I get the following compilation error in 11_1_6.

```
>> Compiling  /afs/cern.ch/work/s/sobhatta/private/HGCal_ele-reco/cmssw_pullreq/ID_var/ShowerShapeVars/CMSSW_11_1_6/src/RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc 
/afs/cern.ch/work/s/sobhatta/private/HGCal_ele-reco/cmssw_pullreq/ID_var/ShowerShapeVars/CMSSW_11_1_6/src/RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc: In constructor 'HGCalShowerShapeHelper::HGCalShowerShapeHelper(edm::ConsumesCollector&)':
/afs/cern.ch/work/s/sobhatta/private/HGCal_ele-reco/cmssw_pullreq/ID_var/ShowerShapeVars/CMSSW_11_1_6/src/RecoEgamma/EgammaTools/src/HGCalShowerShapeHelper.cc:7:43: error: no matching function for call to 'edm::ConsumesCollector::esConsumes()'
     : caloGeometryToken_{sumes.esConsumes()} {}
```

Reply:
can you check against existing examples in 11_1_X, e.g. this one:
https://github.com/cms-sw/cmssw/blob/CMSSW_11_1_X/RecoEgamma/EgammaHLTProducers/plugins/HLTCaloObjInRegionsProducer.cc